helm/mds/templates/configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mds-config-files
+  namespace: {{ .Release.Namespace }}
+data:
+{{ (.Files.Glob "mds-config/*.json").AsConfig | indent 2 }}
```

Comment:
@invertigo How to we update this section or the glob pattern to include `.json` and `.json5` extensions.